dev-requirements.txt

```diff
@@ -5,20 +5,22 @@ wheel==0.30.0
 tornado==5.0.2
 PySocks==1.6.8
 pkginfo==1.4.2
-pytest-random-order==1.0.4
+pytest-random-order==1.0.4;python_version>="3.5"
 pytest-timeout==1.3.1
 pytest==4.0.1
 pytest-cov==2.6.0
 tox==2.9.1
 h11==0.8.0
 cryptography==2.2.1
-trio==0.3.0
+# https://github.com/GoogleCloudPlatform/python-repo-tools/issues/23
+pylint<2.0;python_version<="2.7"
+gcp-devrel-py-tools==0.0.15
+
+# optional dependencies, only intended for use with Python 3.5+
```

Comment:
Let's group optional dependencies, only intended for use with Python 3.5+ similarly for python 2

Reply:
I don't understand, what are you suggesting? The Python 3.5+ dependencies are trio and twisted and are grouped below